tools/buildsys/src/builder.rs

```diff
@@ -140,7 +159,21 @@ fn build(target: &str, build_args: &str, tag: &str, output: &str) -> Result<()>
     let _ = docker(&rmi);
 
     // Build the image, which builds the artifacts we want.
-    docker(&build)?;
+    // HACK: work around a transient, known failure case with Docker.
```

Comment:
Is it worth adding this capability to the `docker` function?  I could imagine a signature like:

```
fn docker(args: &[String], retry: Retry) {...}

enum Retry {
   No,
   Yes {
      count: NonZeroU8,
      retry_failure_messages: &[&str],
   }
}
```

Reply:
That would be neat, not sure if it's worth it right now. Maybe when we have more than one known error?

Reply:
I think my concern was more around the repeated code and the need for a "hack" than the list of strings specifically, but obviously I don't think it's a blocker.  It'll be nice to get around this issue.